x-pack/plugins/enterprise_search/public/applications/shared/doc_links/doc_links.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DocLinksStart } from 'kibana/public';
+
+class DocLinks {
+  public enterpriseSearchBase: string;
+  public appSearchBase: string;
+  public workplaceSearchBase: string;
+  public cloudBase: string;
+
+  constructor() {
+    this.enterpriseSearchBase = '';
+    this.appSearchBase = '';
+    this.workplaceSearchBase = '';
+    this.cloudBase = '';
+  }
+
+  public setDocLinks(docLinks: DocLinksStart): void {
+    this.enterpriseSearchBase = docLinks.links.enterpriseSearch.base;
+    this.appSearchBase = docLinks.links.enterpriseSearch.appSearchBase;
+    this.workplaceSearchBase = docLinks.links.enterpriseSearch.workplaceSearchBase;
+    this.cloudBase = `${docLinks.ELASTIC_WEBSITE_URL}guide/en/cloud/current`;
```

Comment:
[not a blocker, just leaving this comment here before I forget it] It would be awesome to directly add `cloudBase` into the Kibana core helper someday! I'm sure other plugins might need it at some point